### DISCUSSION TITLE
Any plans to introduce granularity in seconds for start time?

### DISCUSSION THREAD

**Discussion by munklarsen** [jhuckaby/Cronicle#566] [Ideas]
Hi,

First off, excellent tools you've built. Running ~25K jobs/day on a clustered setup with 4 nodes. Works great!

Secondly, I have a lot of different tasks that runs every minute and while it is not heavy for cronicle to have them run as such, the servers that actually do the processing have a tendency to be loaded for the first 10-20 seconds of each minute and then do almost nothing for the last 40-50 seconds. Are there any plans to introduce either a random wait (in seconds) to each job/task or the ability to define a start time down to the second. 

I'm looking at my tasks thinking that I could easily automatically sub-devide them into 60 seconds of each minute in order to decrease load on the servers do the actual processing :)

**Comment by jhuckaby** [comment]
Thank you!  That's a large install!  I'm glad it's working well for you.

The answer is yes!  Cronicle v2 (Orchestra) supports a "Delay" timing feature that can delay any event by N seconds.  It will be released this year (2023).

**Comment by mikeTWC1984** [comment]
This could be a good idea for a plugin also. Like launching a group of events in batched with delay. 
Or maybe even "chain many" feature, which could have batch/delay parameters too.
